docs/development/contributing.rst

```diff
@@ -75,18 +75,20 @@ If you would like to contribute on a regular basis, you'll want to read the
 development environment to preview changes, run tests locally, etc.
 
 
-Writers
+Technical Writers
 ~~~~~~~
```

Comment:
Restructured Text (the format the docs are written in) is finicky about the way you declare headlines. The text underneath the headline (the ` ~~~~~~~` above) has to be the same length as the headline itself.